src/ui/screens/contact/ContactGrid.tsx

```diff
@@ -10,14 +10,15 @@ import { getFeedImage } from '../../../helpers/feedHelpers';
 import { defaultImages } from '../../../defaultImages';
 import { MediumText } from '../../misc/text';
 import { TabBarPlaceholder } from '../../misc/TabBarPlaceholder';
+import { Feed } from '../../../models/Feed';
 
 export interface DispatchProps {
-    onPressFeed: (feed: ContactFeed) => void;
+    onPressFeed: (feed: ContactFeed | Feed) => void;
 }
 
 export interface FeedSection {
     title?: string;
-    data: ContactFeed[];
+    data: Array<Feed | ContactFeed>;
```

Comment:
Why this definition has changed? On the ContactGrid we can only display `ContactFeed`s, not ordinary `Feed`s.